modules/machine-user-infra-machines-ibm-z.adoc

```diff
@@ -0,0 +1,148 @@
+
+. You can validate that the Ignition file is available on the URL. The following example gets the Ignition config file for the compute node:
++
+[source,terminal]
+----
+$ curl -k http://<HTTP_server>/worker.ign
+----
+
+. Download the {op-system-base} live kernel, initramfs, and rootfs files by running to following commands:
++
+[source,terminal]
+----
+$ curl -LO $(oc -n openshift-machine-config-operator get configmap/coreos-bootimages -o jsonpath='{.data.stream}' \
+| jq -r '.architectures.s390x.artifacts.metal.formats.pxe.kernel.location')
+----
++
+[source,terminal]
+----
+$ curl -LO $(oc -n openshift-machine-config-operator get configmap/coreos-bootimages -o jsonpath='{.data.stream}' \
+| jq -r '.architectures.s390x.artifacts.metal.formats.pxe.initramfs.location')
+----
++
+[source,terminal]
+----
+$ curl -LO $(oc -n openshift-machine-config-operator get configmap/coreos-bootimages -o jsonpath='{.data.stream}' \
+| jq -r '.architectures.s390x.artifacts.metal.formats.pxe.rootfs.location')
+----
+
+. Move the downloaded {op-system-base} live kernel, initramfs, and rootfs files to an HTTP or HTTPS server that is accessible from the z/VM guest you want to add.
+
+. Create a parameter file for the z/VM guest. The following parameters are specific for the virtual machine:
+** Optional: To specify a static IP address, add an `ip=` parameter with the following entries, separated by colons (“:”):
+... The IP address for the machine.
+... An empty string.
+... The gateway.
+... The netmask.
+... The machine host and domain name in the form `hostname.domainname`. Omit this value to let {op-system} decide.
+... The network interface name. Omit this value to let {op-system} decide.
+... The value `none`.
+** For `coreos.inst.ignition_url=`, specify the URL to the `worker.ign` file. Only HTTP and HTTPS protocols are supported.
+** For `coreos.live.rootfs_url=`, specify the matching rootfs artifact for the kernel and initramfs you are booting. Only HTTP and HTTPS protocols are supported.
+
+** For installations on DASD-type disks, complete the following tasks:
+... For `coreos.inst.install_dev=`, specify `/dev/dasda`.
+... Use `rd.dasd=` to specify the DASD where {op-system} is to be installed.
+... Leave all other parameters unchanged.
++
+Example parameter file, `additional-worker-dasd.parm`:
++
+[source,terminal]
+----
+rd.neednet=1 \
+console=ttysclp0 \
+coreos.inst.install_dev=/dev/dasda \
+coreos.live.rootfs_url=http://cl1.provide.example.com:8080/assets/rhcos-live-rootfs.s390x.img \
+coreos.inst.ignition_url=http://cl1.provide.example.com:8080/ignition/worker.ign \
+ip=172.18.78.2::172.18.78.1:255.255.255.0:::none nameserver=172.18.78.1 \
+rd.znet=qeth,0.0.bdf0,0.0.bdf1,0.0.bdf2,layer2=1,portno=0 \
+zfcp.allow_lun_scan=0 \
+rd.dasd=0.0.3490
+----
++
+Write all options in the parameter file as a single line and make sure you have no newline characters.
```

Comment:
```suggestion
Write all options in the parameter file as a single line and make sure that there are no newline characters.
```

Reply:
Would like to keep the whole sentence in active voice.